packages/zk-transfer-web/src/App.tsx

```diff
@@ -28,16 +29,24 @@ function App({ demo }: AppProp) {
     if (!state.chainId)
       demo.clientInit().then(async () => {
         console.groupCollapsed("Client Initialized");
-        const { zkTokens, thisAddressNotes } = await makeApi(demo);
+        const { zkTokens, thisAddressNotes, bm } = await makeApi(demo);
+
+        // Auto-Mint
+        console.info('Auto-Minting')
+        makeMint(demo, { bm, tradeSymbol: 'AAA', amount: 2 })
```

Comment:
The amount according to the bounty should be `10`, and let's use a `BN` type for the amount consistently everywhere b/c in the future, we should be able to call `makeMint` with the full range of ERC20/ERC1724 note values (potentially 10^18).

Also `demo` is probably better as a member of the object along with `bm`, `tradeSymbol`, and `amount`. Thoughts?

Reply:
Okay.